bciers/apps/reporting/src/app/components/signOff/SignOffPage.tsx

```diff
@@ -53,6 +46,14 @@ export default function SignOffPage({ version_id }: HasReportVersion) {
 
   const handleSubmit = async () => {
```

Comment:
**Suggestion:** Refactored the `handleSubmit` function to set setSubmitButtonDisabled on error and on submit.

Reply:
I'd argue that it would prevent them from re-submitting without unclicking and clicking some checkboxes, if someone else uploaded the file in the meanwhile.
I think we can safely let a user retry, since the error was thrown by the server and not by the current page